src/Services/SearchService.php

```diff
@@ -263,10 +268,18 @@ public function search(HttpRequest $request, ExternalSearch $externalSearch)
      */
     public function aliveTest()
     {
-        $request = $this->requestBuilder->buildAliveRequest();
-        $response = $this->client->call($request);
+        if ($this->aliveTestResult === null) {
+            $request = $this->requestBuilder->buildAliveRequest();
+            $response = $this->client->call($request);
+
+            $this->aliveTestResult = ($response === Plugin::API_ALIVE_RESPONSE_BODY);
+
+            if (!$this->aliveTestResult) {
+                $this->logger->error('Findologic search is not available!');
```

Comment:
@RimantasAdeoWeb 
One point has not been migrated from PLENTY-303 to PLENTY-324, I've done that now. Please update the message and the log level within this PR.

![Screen Shot 2021-04-09 at 13 34 24](https://user-images.githubusercontent.com/511559/114174035-58aec880-9938-11eb-8ed1-547d60ffa87c.png)